app/src/main/res/values/strings.xml

```diff
@@ -96,7 +96,7 @@
     <string name="pref_viewer">Viewer</string>
     <string name="pref_viewer_summary">Picture-in-Picture, Zoom</string>
     <string name="pref_input">Input</string>
-    <string name="pref_input_summary">Gestures, Mouse, Virtual keys</string>
+    <string name="pref_input_summary">Gestures, Mouse, Virtual keys, Key Mappings</string>
```

Comment:
Mentioning 'Key Mapping' is not necessary here. It causes the summery to wrap-around to second line on some smaller devices. It also makes too long with respect to other summaries.